server/datastore/mysql/hosts.go

```diff
@@ -467,6 +467,10 @@ func (ds *Datastore) ListHosts(ctx context.Context, filter fleet.TeamFilter, opt
 }
 
 func (ds *Datastore) applyHostFilters(opt fleet.HostListOptions, sql string, filter fleet.TeamFilter, params []interface{}) (string, []interface{}) {
+	if opt.OSNameFilter != nil && opt.OSVersionFilter != nil {
+		level.Error(ds.logger).Log("err", fmt.Sprintf("name %s version %s", *opt.OSNameFilter, *opt.OSVersionFilter))
```

Comment:
Maybe this left behind from a prev. debugging session?

Reply:
Yes, thanks!